lib/statesman/adapters/sequel.rb

```diff
@@ -0,0 +1,81 @@
+require_relative "../exceptions"
+
+module Statesman
+  module Adapters
+    class Sequel
+      attr_reader :transition_class, :parent_model, :observer
+
+      ::Sequel.extension(:inflector)
+
+      def initialize(transition_class, parent_model, observer)
```

Comment:
This method now requires an additional `options={}` argument to work on the latest version.